scheds/rust/scx_rusty/src/main.rs

```diff
@@ -314,7 +314,6 @@ impl<'a> Scheduler<'a> {
         skel.rodata_mut().load_half_life = (opts.load_half_life * 1000000000.0) as u32;
         skel.rodata_mut().kthreads_local = opts.kthreads_local;
         skel.rodata_mut().fifo_sched = opts.fifo_sched;
-        skel.rodata_mut().switch_partial = opts.partial;
```

Comment:
Ditto for scx_rusty about `--partial`.

Reply:
Ditto.

Reply:
Ah, I was missing this part, that's right. So everything looks good, thanks!